tests/conftest.py

```diff
@@ -246,6 +249,50 @@ def pytest_configure(config):
     )
 
 
+@pytest.fixture
+def sbatch_animation_fc(fc):
+    import functools
+    fc.sr_animation_run = functools.partial(
```

Comment:
Not sure I like this. My bad for starting this trend. We could just add sr_animation_run to srunit. conftest.py seems like the wrong place. sbatch_animation_fc doesn't need to exist, because it should be a mode on all runs in the test file, that is, job_run_mode would be configured with flask_client and be an attribute on fc.